lib/taskana-core/src/main/java/pro/taskana/impl/TaskServiceImpl.java

```diff
@@ -602,7 +606,6 @@ private void removeSingleTask(BulkOperationResults<String, TaskanaException> bul
             LOGGER.debug("exit from updateTasks().");
```

Comment:
The LOGGER statement on line 599: 
LOGGER.debug("updateTasks() updated the following tasks: {} ",  LoggerUtils.listToString(changedTasks));
should only be called if debug is enabled (to invoke listToString() only when required...

Reply:
Done